qiskit/providers/aer/noise/passes/relaxation_noise_pass.py

```diff
@@ -0,0 +1,105 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2021.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+"""
+Thermal relaxation noise pass.
+"""
+from typing import Optional, Union, Sequence, List
+
+import numpy as np
+
+from qiskit.circuit import Instruction, QuantumCircuit
+from qiskit.dagcircuit import DAGCircuit
+from qiskit.transpiler.exceptions import TranspilerError
+from .local_noise_pass import LocalNoisePass
+from ..errors.standard_errors import thermal_relaxation_error
+
+
+class RelaxationNoisePass(LocalNoisePass):
+    """Add duration dependent thermal relaxation noise after instructions."""
+
+    def __init__(
+            self,
+            t1s: List[float],
+            t2s: List[float],
+            dt: float,
+            ops: Optional[Union[Instruction, Sequence[Instruction]]] = None,
+            excited_state_populations: Optional[List[float]] = None,
+    ):
+        """Initialize RelaxationNoisePass.
+
+        Args:
+            t1s: List of T1 times in seconds for each qubit.
+            t2s: List of T2 times in seconds for each qubit.
+            dt: ...
+            ops: Optional, the operations to add relaxation to. If None
+                 relaxation will be added to all operations.
+            excited_state_populations: Optional, list of excited state populations
+                for each qubit at thermal equilibrium. If not supplied or obtained
+                from the backend this will be set to 0 for each qubit.
+        """
+        self._t1s = np.asarray(t1s)
+        self._t2s = np.asarray(t2s)
+        if excited_state_populations is not None:
+            self._p1s = np.asarray(excited_state_populations)
+        else:
+            self._p1s = np.zeros(len(t1s))
+        self._dt = dt
+        super().__init__(self._thermal_relaxation_error, ops=ops, method="append")
+
+    def _thermal_relaxation_error(
+            self,
+            op: Instruction,
+            qubits: Sequence[int]
+    ):
+        """Return thermal relaxation error on each gate qubit"""
+        duration = op.duration
+        if duration == 0:
```

Comment:
```suggestion
        if not duration:
```
So duration = None also returns None